virtual_ecosystem/example_data/config/data_config.toml

```diff
@@ -106,6 +106,21 @@ var_name = "c_n_ratio_below_metabolic"
 [[core.data.variable]]
 file = "../data/example_litter_data.nc"
 var_name = "c_n_ratio_below_structural"
+[[core.data.variable]]
+file = "../data/example_litter_data.nc"
```

Comment:
This is just a generic question and nothing to do with this PR in particular. For what I can see, running a full Virtual Ecosystem simulation requires A LOT of data to be available. Is that data typically available or easy to find? Are you planning to provide defaults for them - some, at least? If the data can be downloaded from online repositories (for free), would you be including hooks to download that data in the background?

Reply:
Yeah this is a good question. In general, a lot of this data will not be particularly easy to get, partly because ecological data tends to get reported in fairly in consistent ways. But also because (at least in the soil and litter models) some variables are chosen because they make sense in the model context rather than being commonly measured empirically, e.g. structural litter isn't measured empirically, mass of dead leaves, twigs, branches etc is.

A few more (Malaysia based) postdocs have joined the team and their role will be getting the data needed for parameterisation, calibration and validation together. I'm not sure of the exact planned workflow there, but I believe the idea is to have all the scripts used to wrangle the data and the final datasets ready for input available online. So this probably make sense to be hooked in some way to the model itself, particularly for the non-site specific data.